.travis.yml

```diff
@@ -5,5 +5,10 @@ jdk:
 
 script:
 - ./gradlew clean build test
-- if [ "$TRAVIS_BRANCH" = "master" ]; then ./gradlew :helios-benchmarks:updateMasterBenchmark; fi
 - ./gradlew :helios-docs:runAnk
+- ./gradlew :helios-benchmarks:executeMasterBenchmark
```

Comment:
@AdrianRaFo IMO we should manually push the current master benchmarks to master in a different PR and then we could avoid this step. The uploaded benchmarks will be the latest master benchmarks. Does it make sense?

Reply:
solved here https://github.com/47deg/helios/pull/45